From e4d6678a1d70c341a5cb56f631e374d542294913 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 9 Dec 2008 13:06:20 +0800
Subject: [PATCH] ENGR00101930: fix pmic compile fail

Any one of the MC13892 PMIC config options
are not turned on in the build option, MX37
build breaks.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 include/asm-arm/arch-mxc/pmic_external.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/asm-arm/arch-mxc/pmic_external.h b/include/asm-arm/arch-mxc/pmic_external.h
index 464168a..7ba9f26 100644
--- a/include/asm-arm/arch-mxc/pmic_external.h
+++ b/include/asm-arm/arch-mxc/pmic_external.h
@@ -1114,6 +1114,7 @@ typedef enum {
 /* EXPORTED FUNCTIONS */
 #ifdef __KERNEL__
 
+#if defined(CONFIG_MXC_PMIC)
 /*!
  * This function is used to determine the PMIC type and its revision.
  *
@@ -1225,7 +1226,7 @@ static inline int reg_mc13892_probe(void)
 #ifdef CONFIG_REGULATOR_MC34704
 int reg_mc34704_probe(void);
 #endif
-
+#endif				/*CONFIG_MXC_PMIC*/
 #endif				/* __KERNEL__ */
 /* CONFIG_MXC_PMIC_MC13783 || CONFIG_MXC_PMIC_MC9SDZ60 */
 
-- 
1.5.4.4

